### PR TITLE
Hts 325

### PR DESCRIPTION
**Pull request by Jackie1988**

### CODE REVIEW

**Comment by sureshhmrc** [comment]
please write unit tests for this change, 
Also run the coverage locally to make sure we reach required level of coverage

File: app/uk/gov/hmrc/helptosave/connectors/UserInfoAPIConnector.scala [hunk 1]
```diff
@@ -84,10 +85,23 @@ class UserInfoAPIConnectorImpl @Inject()(configuration: Configuration, ec: Execu
     })
   }
 
+  private def returnUserInfoWithCC(userInfo: OpenIDConnectUserInfo): OpenIDConnectUserInfo = {
+    val addressWithCountry : Option[Address] =  userInfo.address match {
+      case Some(address) ⇒
+        val code1 = address.code match {
```
Comment (sureshhmrc) [inline app/uk/gov/hmrc/helptosave/connectors/UserInfoAPIConnector.scala]: `code1` could be renamed to something more proper ?

**Comment by Jackie1988** [comment]
I have written a unit test for this change and implemented the required functionality in less code.

**Comment by Jackie1988** [comment]
Changes no longer needed.

**Pull request closed without merging**